Week-5/assignment-3/index.js

```diff
@@ -0,0 +1,16 @@
+
+const myArray = [1, 2, 3, 4, 5, 1, 4, 6, 3];
+const myArray2 = [1, 2, 3, 4, 5];
+
+function hasDuplicates(arr) {
+    let mySet = new Set(arr);
```

Comment:
do we need to create the whole set. Cant we fail early? Think about the time and space complexity. In your code the best case is O(n).

Reply:
everything looks good to me here , will ask this on 1-1